From 0b390942a750ad06a8e454ddc677be1a84aa3e1a Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Wed, 25 Jul 2018 18:53:28 +0000
Subject: [PATCH] TRITON-587 net-agent needs updated regex for rack tagged
 admin nics (bump version) Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Approved by: Cody Peter Mello <cody.mello@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index c16fdea..e29dd92 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "net-agent",
     "description": "Triton Network Agent",
-    "version": "2.0.0",
+    "version": "2.1.0",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
-- 
2.21.0

